dom-manipulation/main.js

```diff
@@ -0,0 +1,24 @@
+let clicks = 0;
+
+const hotButton = document.querySelector('.hot-button');
+const clickCount = document.querySelector('.click-count');
+
+hotButton.addEventListener('click', function (event) {
+  clicks++;
+  if (clicks < 4) {
+    hotButton.className = 'hot-button cold';
+  } else if (clicks < 7) {
+    hotButton.className = 'hot-button cool';
+  } else if (clicks < 10) {
+    hotButton.className = 'hot-button tepid';
+  } else if (clicks < 13) {
+    hotButton.className = 'hot-button warm';
+  } else if (clicks < 16) {
+    hotButton.className = 'hot-button hot';
+  } else {
+    hotButton.className = 'nuclear!';
+  }
+
+  console.log(`Button clicked ${clicks} times.`);
```

Comment:
Unless we specifically ask for them in the instructions, you should remove all `console.log`s before submitting your PR. They are only for testing / debugging purposes